README.md

```diff
@@ -32,27 +35,27 @@ Sequential:
 /usr/bin/time make sequential  169.32s user 6.59s system 100% cpu 2:55.65 total
 ```
 
-Parallel w/4 jobs:
+Parallel chunks w/4 jobs:
 
 ```
 164.22user 6.89system 0:47.53elapsed 359%CPU (0avgtext+0avgdata 192548maxresident)k
 0inputs+1274824outputs (0major+3642408minor)pagefaults 0swaps
-/usr/bin/time make parallel NUMBER_OF_THREADS=4  164.22s user 6.90s system 359% cpu 47.536 total
+/usr/bin/time make parallel-chunks NUMBER_OF_THREADS=4  164.22s user 6.90s system 359% cpu 47.536 total
 ```
 
-Parallel w/8 jobs
+Parallel chunks w/8 jobs
 
 ```
 226.84user 9.27system 0:34.18elapsed 690%CPU (0avgtext+0avgdata 191928maxresident)k
 0inputs+1274856outputs (0major+3827022minor)pagefaults 0swaps
-/usr/bin/time make parallel NUMBER_OF_THREADS=8  226.85s user 9.28s system 690% cpu 34.188 total  
+/usr/bin/time make parallel-chunks NUMBER_OF_THREADS=8  226.85s user 9.28s system 690% cpu 34.188 total
 ```
 
-Parallel w/16 jobs (sanity check)
+Parallel chunks w/16 jobs (sanity check)
 
 ```
 252.12user 11.21system 0:35.84elapsed 734%CPU (0avgtext+0avgdata 191968maxresident)k
 400inputs+1274920outputs (19major+4210729minor)pagefaults 0swaps
-/usr/bin/time make parallel NUMBER_OF_THREADS=16  252.13s user 11.21s system 734% cpu 35.850 total 
+/usr/bin/time make parallel-chunks NUMBER_OF_THREADS=16  252.13s user 11.21s system 734% cpu 35.850 total
```

Comment:
Here's the `/usr/bin/time make parallel NUMBER_OF_THREADS=8` output for comparison

```suggestion
528.31user 47.80system 1:21.19elapsed 709%CPU (0avgtext+0avgdata 160924maxresident)k
0inputs+1303544outputs (0major+10202132minor)pagefaults 0swaps
/usr/bin/time make parallel  528.32s user 47.80s system 709% cpu 1:21.20 total
```

Reply:
Thanks for measuring! The suggsted diff looks strange, though...

Reply:
Yeah, sry, that should not have been a suggestion, I'll add that to the README after merge.